src/include/buffer_raw.h

```diff
@@ -92,12 +92,9 @@ namespace ceph::buffer {
     virtual int zero_copy_to_fd(int fd, loff_t *offset) {
       return -ENOTSUP;
     }
-    virtual bool is_page_aligned() {
+    virtual bool is_page_aligned() final {
```

Comment:
Marked as `final` solely for testing. Next commit will strip `virtual`-ness from this method.